reliability-v2/config/example_reliability.yaml

```diff
@@ -44,6 +44,7 @@ reliability:
       tasks:
         - func delete_all_projects # clear all projects
         - func new_project 2 # new 2 projects
+        #- func apply 2 "<path to /reliability-v2/networkpolicy/allow-same-namespace.yaml>" # Apply network policy to 2 projects
```

Comment:
do you want to have this commented out?

Reply:
@paigerube14 Yeah I intended to do this, as not all the tests needs network policy. I added a comment above this line to explain that this line is valid but only use it when you want to include network policy in the test.